package.json

```diff
@@ -55,7 +55,7 @@
     "tonic-example.js"
   ],
   "dependencies": {
-    "contentful-sdk-core": "^6.4.5",
+    "contentful-sdk-core": "^6.6.0",
     "axios": "^0.21.0",
```

Comment:
Possible to remove `axios` as direct dependency?

Reply:
no, it's still used in a couple places